kernel-boot/rdma_rename.c

```diff
@@ -629,6 +650,8 @@ int main(int argc, char **argv)
 		ret = by_pci(&d);
 	if (ret && (np & NAME_GUID))
 		ret = by_guid(&d);
+	if (ret && (np & NAME_FIXED))
```

Comment:
It is not correct if (..), if everything above failed, we will execute this section which is not what we want.

You need to place NAME_FIXED outside of this if .. block.

Reply:
I was just trying to keep with the convention of the current code.  NAME_FIXED can only be set if the other options aren't set (in the current implementation).  By "and"ing with the return value we are just setting an order of precedence if this ever were to change.  Would you prefer it to be, `if (np & NAME_FIXED)`?  (it should be checked for the `if(ret) goto out;` part)

Reply:
Please squash all commits into one and push with "-f" to this topic.

The "ret ... " scheme is needed for the fall-back, but you are right, we can leave it here.
Sorry for my mistake.

Reply:
I've squashed the commits.  Thanks

Reply:
1. Your commit message is still empty.
2. Update and add an example to kernel-boot/rdma-persistent-naming.rules as a comment, something like this:
#ACTION=="add", SUBSYSTEM=="infiniband", PROGRAM="rdma_rename %k NAME_FIXED static_name"

Reply:
I've updated both.  Thanks!